tests/fixtures/1.29.2/.gitignore

```diff
@@ -1,3 +1,3 @@
 /target
 **/*.rs.bk
-!Cargo.lock
\ No newline at end of file
+Cargo.lock
```

Comment:
Running tests locally was showing this file as untracked, so fixed this since I assume it's not wanted in the repository

Reply:
This is an unfortunate thing. I think it needs to not be in a gitingore (because the lockfile will be overwritten by the test, and should not be committed for the case in particular, because it needs to be regenerated on each cargo msrv iteration, and keep the original file in place). At some point I need to improve the test case, e.g. by using a tempfile. This is a not great test case as written on my part.